scripts/code_review_tool_runner.py

```diff
@@ -30,7 +30,10 @@ def run(args) -> None:
         show_patch_example=False,
     )
 
-    review_data = code_review.review_data_classes[args.review_platform]()
+    if args.review_platform == "phabricator":
+        review_data = code_review.PhabricatorReviewData()
```

Comment:
What was the problem before this change?

Reply:
mypy complained that we cannot instantiate the `ReviewData` class since it is abstract. 

It could be a bug with mypy, since we are not instantiating `ReviewData`, but it is the parent for the classes that we might instantiate.